internal/captain/command.go

```diff
@@ -172,6 +172,10 @@ func (c *Command) SetDeferAnalytics(value bool) {
 	c.deferAnalytics = value
 }
 
+func (c *Command) SetHidden(value bool) {
```

Comment:
We seem to already have something for this:

http://github.com/ActiveState/cli/blob/77e2701270db9b9789e00e1274af701a5fa1443c/internal/captain/option.go#L7-L7

Reply:
Apologies if this is obvious; Where do the options get set/processed?

Reply:
Hmm looks like it got dropped along the line somewhere, and I can't find where it originated from. Ok disregard my comment here. Feel free to remove that file if you like, seems both functions are unused.